app/views/donations/show.html.erb

```diff
@@ -58,6 +58,7 @@
     <div class="box-footer">
       <%= edit_button_to edit_donation_path(@donation), { text: "Make a correction", size: "lg" } %>
       <%= new_button_to new_distribution_path(donation_id: @donation.id, storage_location_id: @donation.storage_location_id), { text: "Start a new Distribution" } %>
+      <%= delete_button_to donation_path(@donation), { size: "lg", confirm: "Are you sure you want to permanently remove this donation?" } if current_user.organization_admin? %>
```

Comment:
Thanks for using the `UiHelper` module! I was specifically wanting to make sure you had used it :)